### PR TITLE
chore(server): Update response types following OrderFilter

### PR DESCRIPTION
**Pull request by Mitsichury**

### CODE REVIEW

File: tools/types/src/server.ts [hunk 1]
```diff
@@ -22,28 +23,20 @@ export enum SortOrder {
...
   senderTokens?: string[]
+  senderWallet?: string
+  signerMaxAmount?: string
+  signerMinAmount?: string
+  signerTokens?: string[]
+  signerWallet?: string
   sortField?: SortField
   sortOrder?: SortOrder
-  offset: number
+  tokenIds?: string[]
```
Comment (dmosites) [inline tools/types/src/server.ts:37]: Probably want to split this into `signerIds` and `senderIds` as is done with the signer and sender tokens.

File: tools/types/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@airswap/types",
-  "version": "4.0.4",
+  "version": "4.1.0",
```
Comment (dmosites) [inline tools/types/package.json:3]: We're still iterating on the minor version (and building the only clients using it so far) so let's just bump this to `4.0.5`.

File: tools/types/src/server.ts (hunk 1 above)
Comment (Mitsichury) [reply tools/types/src/server.ts]: done

File: tools/types/package.json (hunk 2 above)
Comment (Mitsichury) [reply tools/types/package.json]: done

File: tools/types/src/server.ts [hunk 3]
```diff
@@ -55,13 +49,10 @@ export type IndexedOrder<Type> = {
...
 }
 
 export type Pagination = {
-  first: string
-  last: string
-  prev?: string | undefined
-  next?: string | undefined
+  limit: number
+  offset: number
+  totalForQuery: number
```
Comment (piersss) [inline tools/types/src/server.ts:57]:
```suggestion
  total: number
```

I think Don said `total`. 😁

**Review by piersss — APPROVED** [review]
LGTM

**Pull request merged**